code/game/objects/machinery/computer/dropship_picker.dm

```diff
@@ -0,0 +1,76 @@
+
+/obj/machinery/computer/dropship_picker
+	name = "dropship picker"
+	desc = "A computer that lets you choose the model of the tadpole.."
+	density = TRUE
+	icon = 'icons/obj/machines/computer.dmi'
+	icon_state = "computer_generic"
+	circuit = null
+	resistance_flags = RESIST_ALL
+	interaction_flags = INTERACT_MACHINE_TGUI
+	req_access = list(ACCESS_MARINE_DROPSHIP)
+	var/dock_id = SHUTTLE_TADPOLE
+	///if true lock console
+	var/dropship_selected = FALSE
+	var/datum/map_template/shuttle/current_template_ref
```

Comment:
You cast this then proceed to use locate and refs just set it after locating
also missing autodoc